whois-update/src/main/java/net/ripe/db/whois/update/authentication/strategy/OrgRefAuthentication.java

```diff
@@ -1,38 +1,23 @@
 package net.ripe.db.whois.update.authentication.strategy;
 
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Sets;
-import net.ripe.db.whois.common.Message;
 import net.ripe.db.whois.common.dao.RpslObjectDao;
-import net.ripe.db.whois.common.domain.CIString;
 import net.ripe.db.whois.common.rpsl.AttributeType;
 import net.ripe.db.whois.common.rpsl.ObjectType;
 import net.ripe.db.whois.common.rpsl.RpslObject;
 import net.ripe.db.whois.update.authentication.credential.AuthenticationModule;
 import net.ripe.db.whois.update.domain.PreparedUpdate;
 import net.ripe.db.whois.update.domain.UpdateContext;
-import net.ripe.db.whois.update.domain.UpdateMessages;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.stereotype.Component;
 
-import java.util.Collection;
-import java.util.LinkedHashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.Set;
 
 @Component
-public class OrgRefAuthentication extends AuthenticationStrategyBase {
-
-    private final AuthenticationModule credentialValidators;
-    private final RpslObjectDao rpslObjectDao;
+public class OrgRefAuthentication extends ReferenceAuthentication {
```

Comment:
Can you combine all `mnt-ref` authentication into *one* class, e.g. `MntRefAuthentication` rather than using a separate class per object type?

Reply:
Done